src/Wpf.Ui.Gallery/Views/Windows/MainWindow.xaml

```diff
@@ -3,6 +3,8 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:iconElements="clr-namespace:Wpf.Ui.Controls.IconElements;assembly=Wpf.Ui"
```

Comment:
`ImageIcon` and `IconSourceElement` should be in **xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"** namespace